src/components/ui/Navigation.vue

```diff
@@ -16,7 +16,7 @@
           <router-link to="/">Support</router-link>
         </div>
         <div class="navbar-item connect">
-          <a href="" v-on:click="setAddress">Connect</a>
+          <a href="" v-on:click="setPublicKey">Connect</a>
```

Comment:
Would be nice to be syntactically consistent in using the shorthand `@` instead of `v-on:` throughout the app

Reply:
that's a good catch!